docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -25,7 +25,7 @@ Building a Design with TracerV
 
 In all FireChip designs, TracerV is included by default. Other targets can
 enable it by attaching a TracerV Bridge to the RISC-V trace port of one-or-more
-cores. By default, only the cycle number, instruction address, and valid bit
+cores or instruction streams. By default, only the cycle number, instruction address, and valid bit
```

Comment:
Can we not just call the disjunction here a hart, or am i, too, misunderstanding the term.

Reply:
It is not a "hart". It is the stream of committed instructions. Thus for something like BOOM that commits N>=1 instructions, that value has to be specified here. This gets even tricker when there are multi-core designs where each core has a different amount of commit ports.